src/IO/H5/Python/Dat.cpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <Python.h>
+#include <boost/python.hpp>
+#include <boost/python/stl_iterator.hpp>
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+#include <hdf5.h>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/Matrix.hpp"
+#include "IO/H5/Dat.hpp"
+#include "IO/H5/Header.hpp"
+#include "IO/H5/OpenGroup.hpp"
+#include "IO/H5/Python/Numpy.hpp"
+#include "IO/H5/Python/ToNumpy.hpp"
+#include "IO/H5/Version.hpp"
+#include "PythonBindings/VectorPyList.hpp"
+
+namespace bp = boost::python;
+
+namespace py_bindings {
+void bind_h5dat() {
+  // Wrapper for basic H5Dat operations
+  bp::class_<h5::Dat, boost::noncopyable>("H5Dat", bp::no_init)
+      .def("append",
+           +[](h5::Dat& D, const bp::list& data) {
+             D.append(py_list_to_std_vector<double>(data));
+           })
+      .def("get_legend",
```

Comment:
Use `def_readonly` for these properties instead, and remove the `get_` prefix.

Reply:
I'm not entirely sure, I'm trying to parse the documentation, but it seems that `def_readonly` can only access public members of the class.  I can use `add_property` instead and just not add in setter functions and it should be functionally equivalent.  Any thoughts.

Reply:
That seems fine if indeed the setting raises an error